script.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector(".hamburger");
+const navMenu = document.querySelector(".NavMenu");
+const Close = document.querySelector(".Close");
+
+hamburger.onclick = () => {
+  //call-back function
+  navMenu.classList.add("mike");
+};
+Close.onclick = () => {
+  //call-back function
+  navMenu.classList.remove("mike");
+};
```

Comment:
- [ ] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears that's not happening So please fix that.

![chrome-capture-2022-5-22](https://user-images.githubusercontent.com/68796232/174990177-fd8004b3-739c-4322-a572-2c3cbb6a6e71.gif)